docs/0.1/creating_locations.md

```diff
@@ -0,0 +1,376 @@
+`grid` commands in this procedure.
+
+{:start="2"}
+
+2. Set `GRID_DAEMON_KEY` to the base name of your public/private key files
+   (the example environment uses `alpha-agent`).  This environment variable
+   replaces the `-k` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
+   daemon's key files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
+   `01234-ABCDE::gsAA`) for the scabbard service on the circuit. This
+   environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::gsAA"
+   ```
+
+   Tip: See [Prerequisites](#prerequisites) for the `splinter` commands that
+   display the circuit ID and four-character service ID string.
+
+### Define and Create a Location
+
+Each location requires an organization ID (the owner) and at least one agent
+with permission to create, update, and delete the location. There must also
+be an existing location schema for the location's properties.
+See [Prerequisites](#prerequisites) for more information.
+
+{:start="5"}
+
+5. Create a location definition file, in YAML format, that specifies the
+   following information:
+
+   * location namespace (GS1 for this example) that corresponds with the
+     location schema name (`gs1_location` for this example).
+   * Location ID (such as a GTIN)
+   * Owner (organization ID)
+   * Set of location attributes (each with a name and value that conforms to
+     the data type)
+     <br><br>
+
+    ```yaml
+    - namespace: "GS1"
+      location_id: "9012345000004"
+      owner: "314156"
+      properties:
+        locationDescription: "test location"
+        locationType: 0
+        addressLine1: "251 Test St."
+        city: "Minneapolis"
+        stateOrRegion: "MN"
+        postalCode: "55401"
+        country: "United States"
+        latLongValue: "449778,-932650"
+        contactName: "John Tester"
+        contactEmail: "tester@email.com"
+        contactPhone: "6555555555"
+        createDate: 1602706399
+    ```
+
+    Tip: Use a YAML linter to validate the new file is formatted correctly.
+
+1. Use `docker cp` to copy the file into the `gridd-alpha` container.
+
+   ```
+   $ docker cp location.yaml gridd-alpha:/
+   ```
+
+1. Add the new location by using the `grid location create` command to specify the
+   definition in the `location.yaml` file.
+
+   ```
```

Comment:
Before creating the location, the user needs to have permissions.

```
grid \
agent update 013600 $(cat ~/.grid/keys/alpha-agent.pub) --active \
--role can_create_location \
--role can_update_location \
--role can_delete_location \
--role admin
```

Reply:
User creation is covered in a different docs page and the necessary permissions are listed on line 219-220